This patch removes references to unneeded manual pages
that are only relevant for Windows operating system.

The change will not be sent to upstream since it is Solaris specific.

--- rabbitmq-server-3.8.35/deps/rabbit/docs/rabbitmq-env.conf.5
+++ rabbitmq-server-3.8.35/deps/rabbit/docs/rabbitmq-env.conf.5
@@ -74,7 +74,6 @@ file RabbitMQ configuration file locatio
 .\" ------------------------------------------------------------------
 .Sh SEE ALSO
 .\" ------------------------------------------------------------------
-.Xr rabbitmq-echopid 8 ,
 .Xr rabbitmq-plugins 8 ,
 .Xr rabbitmq-server 8 ,
 .Xr rabbitmq-queues 8 ,
--- rabbitmq-server-3.8.35/deps/rabbit/docs/rabbitmq-server.8
+++ rabbitmq-server-3.8.35/deps/rabbit/docs/rabbitmq-server.8
@@ -91,7 +91,6 @@ For example, runs RabbitMQ AMQP server i
 .Xr rabbitmq-upgrade 8 ,
 .Xr rabbitmq-service 8 ,
 .Xr rabbitmq-env.conf 5 ,
-.Xr rabbitmq-echopid 8
 .\" ------------------------------------------------------------------
 .Sh AUTHOR
 .\" ------------------------------------------------------------------
--- rabbitmq-server-3.8.35/deps/rabbit/docs/rabbitmq-plugins.8
+++ rabbitmq-server-3.8.35/deps/rabbit/docs/rabbitmq-plugins.8
@@ -247,7 +247,6 @@ plugin and its dependencies and disables
 .Xr rabbitmq-upgrade 8 ,
 .Xr rabbitmq-service 8 ,
 .Xr rabbitmq-env.conf 5 ,
-.Xr rabbitmq-echopid 8
 .\" ------------------------------------------------------------------
 .Sh AUTHOR
 .\" ------------------------------------------------------------------
--- rabbitmq-server-3.8.35/deps/rabbit/docs/rabbitmqctl.8
+++ rabbitmq-server-3.8.35/deps/rabbit/docs/rabbitmqctl.8
@@ -2496,7 +2496,6 @@ Reset stats database for all nodes in th
 .Xr rabbitmq-upgrade 8 ,
 .Xr rabbitmq-service 8 ,
 .Xr rabbitmq-env.conf 5 ,
-.Xr rabbitmq-echopid 8
 .\" ------------------------------------------------------------------------------------------------
 .Sh AUTHOR
 .\" ------------------------------------------------------------------------------------------------
